app/src/main/java/com/codesoom/assignment/controller/ProductController.java

```diff
@@ -0,0 +1,65 @@
+// TODO: Cotroller List 구현하기 (테스트 코드 통과)
+// TODO: Cotroller Detail 구현하기
+// TODO: Cotroller Create 구현하기
+// TODO: Cotroller update 구현하기
+// TODO: Cotroller patch 구현하기
+// TODO: Cotroller delete 구현하기
+
+package com.codesoom.assignment.controller;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.domain.Product;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/products")
+@CrossOrigin
+public class ProductController {
+    Product product = new Product();
+
+    private List<Product> products = new ArrayList<>();
+    private Long newId = 0L;
+
+    public ProductController(Product product) {
+        this.product = product;
+    }
```

Comment:
컨트롤러가 생성자를 통해 상품을 주입받을 이유가 없어 보여요.
```suggestion
    public ProductController() {
    }
```